### ISSUE TITLE
Proposal for functional tests: save logs on test failure to artifact directory and don't put to std out

### ISSUE THREAD

**Issue by birdsarah** [mozilla-mobile/mozilla-vpn-client#2104]
I'm not sure how much utility there is in the logs being printed to std out in the test run. It slows down the CI rendering and makes it hard to parse what happened. I'd like to propose just writing them to the artifacts directory on fail (which may already happen) and not printing to std out.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-1213)

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

Something similar has been done already. We have logs as artifacts.
